src/styles/blocks/catalog.scss

```
@@ -14,12 +19,18 @@
 
     width: 200px;
     height: 300px;
-    border: 1px solid #ccc;
+    border: 1px solid $color-white;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform 300ms;
+
```

scale 2 - it's too much
```suggestion
    transition: transform $transitionDuration;

    &:hover {
    transform: scale(1.2);
    color: $color-grey;
  }
```